test/context.test.ts

```diff
@@ -97,9 +97,12 @@ describe('Context', () => {
 
   describe('config', () => {
     let github: GitHubAPI
+    type GetContentsReturn = ReturnType<typeof github.repos.getContents>
 
     function responseFromString (content: string) {
```

Comment:
Oooh that's a nice improvement @gr2m - you could also put the return type here, for some DRY ✨ 

```suggestion
    function responseFromString (content: string): GetContentsReturn {
```

Reply:
smart!